### DISCUSSION TITLE
What is the version support for Netbox?

### DISCUSSION THREAD

**Discussion by sndrr** [netbox-community/netbox#14170] [Q&A] [answered]
I searched for a release and support roadmap, but I could not find one.

I know that Netbox releases 3 times a year (April, August) and December, from the [release notes](https://github.com/netbox-community/netbox/blob/develop/docs/release-notes/index.md)), but there is no explicit version support or EOL schedule. Looking back in the release history, I see that as soon as a new minor version is released (out of beta), there are no new patch releases on previous minor versions.

Can I conclude that minor versions are supported until a new minor version is released? And therefore, older versions are then EOL?

**Comment by PaulR282** [comment, marked as answer]
> Can I conclude that minor versions are supported until a new minor version is released? And therefore, older versions are then EOL?

Yes, that is correct.

**Comment by hex-m** [comment]
I was expecting to find that information in the [security tab](https://github.com/netbox-community/netbox/security).
